### PR TITLE
fix clang compilation of inv_cg_quda

### PR DESCRIPTION
**Pull request by mathiaswagner**
array of non-POD is not allowed in C++, only gcc compiles that.

Not sure how to test with Np set.

### CODE REVIEW

File: lib/inv_cg_quda.cpp [hunk 1]
```diff
@@ -399,11 +399,11 @@ namespace quda {
 	  } else {
 
 	    if ( (j+1)%Np == 0 ) {
-	      Complex alpha_[Np];
+	      auto alpha_ = std::unique_ptr<Complex[]>(new Complex[Np]);
```
Comment (maddyscientist) [inline lib/inv_cg_quda.cpp]: Memory leak?

Comment (mathiaswagner) [inline lib/inv_cg_quda.cpp]: Why? It is a unique_ptr.

Comment (maddyscientist) [inline lib/inv_cg_quda.cpp]: you are correct of course 😄

**Comment by maddyscientist** [comment]
To test with Np, you just run invert_test with `--solution-pipeline N`.  And now I see that I failed to add this to the command line help list....  Perhaps you can add that here while testing? 😉

**Comment by mathiaswagner** [comment]
Will do.

**Comment by mathiaswagner** [comment]
`--solution-pipeline N` is already present in line of 1664 of test_util.cpp for me.

**Pull request merged**